nnvm/python/nnvm/frontend/tensorflow.py

```diff
@@ -129,11 +133,17 @@ def _impl(inputs, attr, params):
         else:
             raise TypeError("Unsupported data_format type : {}".format(attr['data_format']))
 
+        if attr['_target_layout'] == "NCHW" and attr['data_format'] == "NHWC":
+            tmp_shape = attr['_input_shapes'][inputs[0]][0]
+            input_shape = [tmp_shape[ii] for ii in (0, 3, 1, 2)]
+            inputs[0] = _sym.transpose(inputs[0], axes=(0, 3, 1, 2))
+            attr['data_format'] = "NCHW"
+            flip_layout = True
+
         # Fix strides
         attr['strides'] = (attr['strides'][1], attr['strides'][2])
```

Comment:
what about this one? does this work for NCHW?

Reply:
Strides also require handling for NCHW. This is handled in my other branch where I am working on NCHW with resnet models.

Reply:
ok